.github/workflows/test-and-release.yml

```diff
@@ -51,6 +52,7 @@ jobs:
           init-args: ${{ matrix.init-args }}
           validate-args: ${{ matrix.validate-args }}
           fmt-args: ${{ matrix.fmt-args }}
+          test-args: ${{ matrix.test-args }}
```

Comment:
still not executed here as test is disabled by default -> move it to the `test-terraform-test` job